### PR TITLE
Add section on ACL Representation and Interpretation

### LINKED ISSUE #172: Clarify algorithm for finding authorizations
Adding this as a placeholder for possible clarification based on resolution of:

  * Algorithm for finding authorizations in an ACL document - https://github.com/solid/web-access-control-spec/issues/19
  * Default ACLs - https://github.com/fcrepo/fcrepo-specification/issues/163

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #172 
(also adjusts positioning of &lt;pre&gt; tags to fix spacing in later examples)

### CODE REVIEW

**Review by escowles — DISMISSED** [review]

**Review by awoods — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -1024,6 +1024,44 @@ <h2>Inheritance and Default ACLs</h2>
...
+      <section id="acl-representation">
+        <h2>ACL Representation and Interpretation</h2
+        <p>
+          ACL resources are represented as described in [[!SOLIDWEBAC]]
+          <a href="https://github.com/solid/web-access-control-spec#representation-format">Representation
+          Format</a>. Implementations MUST inspect the ACL RDF for authorizations. Authorizations are identified by
+          type definition triples of the form <code>authorization_N rdf:type acl:Authorization</code>, where
+          <code>authorization_N</code> is the URI of an authorization. Implementations MUST use only statements
+          associated with an authorization in the ACL RDF to determine access, they MUST NOT dereference the
+          URI of the authorization. The authorizations MUST be examined to see whether they grant the requested
```
Comment (dannylamb) [inline index.html]: What about agentGroup?  The URI would have to be dereferenced then.

Comment (zimeon) [reply index.html]: Well spotted @dannylamb ,  this clearly needs to be addressed so I'll propose a fix

**Comment by csarven** [comment]
Advanced bikeshedding to the rescue:

I find it preferable to avoid whitespace before/after the actual content in pre (unless it is really intended to be there). So, no new lines right after `<pre>` or before `</pre>`. It also reduces any unexpected styling issues.

If pre's text content could be cleaned (with or without RDFa), omitting whitespaces is all around better for potential reuse.

**Comment by zimeon** [comment]
Thanks for your fine bikeshedding @csarven, I will fix this. I think there should be a new line before the closing &lt;/pre&gt;, but no whitespace after opening &lt;pre&gt;

**Comment by zimeon** [comment]
OK, ready for review again. 

I have ended up removing the explicit "MUST NOT dereference the authorization URI" because I think that has always been open to some confusion and is no longer necessary after adding the _"...except in the case of `acl:agentGroup` statements where the group listing document is dereferenced"_ clause.

**Review by escowles — APPROVED** [review]

**Review by awoods — APPROVED** [review]

**Review by dannylamb — APPROVED** [review]

**Pull request merged**